### PR TITLE
Add protractor typescript template

### PR DESCRIPTION
**Pull request by MilesLin**
I added protractor-typescript formatter, which is used for built-in e2e template of Angular default project.

### CODE REVIEW

**Comment by carter-vanekeren** [comment]
This is awesome!  Thank you :)

**Comment by devalex88** [comment]
This is awesome - thank you so much for the PR!

Sorry for the late response, we will merge and release it next week.

**Comment by carter-vanekeren** [comment]
Been using the formatter for a project at work and its been working great so far.  
Hope the feedback is helpful!

File: panel/js/katalon/newformatters/protractorts.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    // https://www.protractortest.org/#/api
+    // katalon
+    // https://docs.katalon.com/katalon-recorder/docs/selenese-selenium-ide-commands-reference.html
+    const seleneseCommands = {
+        "open": "await browser.get('_TARGET_');",
+        "click": "await element(_BY_LOCATOR_).click();",
+        "clickAndWait": 
+            "const el__STEP_ = element(_BY_LOCATOR_);\n" +
+            "\t\tawait browser.wait(EC.elementToBeClickable(el__STEP_));\n" + 
+            "\t\tawait await el__STEP_.click();",
```
Comment (doggy8088) [inline panel/js/katalon/newformatters/protractorts.js]: It's a bug. You should remove one `await` on this line.

Comment (devalex88) [reply panel/js/katalon/newformatters/protractorts.js]: Thank you @doggy8088.

It has been fixed by @minhthanh3145.

**Pull request merged**